sdk/python/feast/infra/materialization/contrib/bytewax/bytewax_materialization_dataflow.py

```diff
@@ -31,14 +30,18 @@ def __init__(
         self.feature_view = feature_view
         self.worker_index = worker_index
         self.paths = paths
+        self.mini_batch_size = int(
+            os.getenv("BYTEWAX_MINI_BATCH_SIZE", DEFAULT_BATCH_SIZE)
+        )
 
         self._run_dataflow()
 
     def process_path(self, path):
+        logger.info(f"Processing path {path}")
```

Comment:
Use lazy formatting in logs (same applies in other places)
```suggestion
        logger.info(f"Processing path %s", path)
```